example/ck_tile/03_gemm/gemm_basic.hpp

```diff
@@ -0,0 +1,80 @@
+
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2024, Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+#include "ck_tile/core.hpp"
+#include "ck_tile/host/kernel_launch.hpp"
+#include "ck_tile/ops/epilogue.hpp"
+#include "ck_tile/ops/gemm.hpp"
+#include <string>
+
+template <typename DataType>
+struct GemmBasicTypeConfig;
+
+template <>
+struct GemmBasicTypeConfig<ck_tile::half_t>
+{
+    using XDataType   = ck_tile::half_t;
+    using YDataType   = ck_tile::half_t;
+    using AccDataType = float;
+    using ODataType   = ck_tile::half_t; // type convert
+    // ToDo: Add more bias config to support different categories of GEMM.
+};
+
+template <ck_tile::MatrixALayout A, ck_tile::MatrixBLayout B, ck_tile::MatrixCLayout C>
+struct LayoutConfig
+{
+    static constexpr ck_tile::MatrixALayout LayoutA = A;
+    static constexpr ck_tile::MatrixBLayout LayoutB = B;
+    static constexpr ck_tile::MatrixCLayout LayoutC = C;
+};
+
+template <typename T>
+struct DataTypeTraits;
+
+template <>
+struct DataTypeTraits<float>
+{
+    static constexpr const char* name = "float";
+};
+
+template <>
+struct DataTypeTraits<double>
+{
+    static constexpr const char* name = "double";
+};
+
+template <>
+struct DataTypeTraits<ck_tile::half_t>
+{
+    static constexpr const char* name = "fp16";
+};
+
+using Types = GemmBasicTypeConfig<ck_tile::half_t>;
+
+// Specific type aliases for easy access
+using XDataType   = Types::XDataType;
+using YDataType   = Types::YDataType;
+using AccDataType = Types::AccDataType;
+using ODataType   = Types::ODataType;
+
+struct gemm_basic_args
+{
+    const void* p_a;
+    const void* p_b;
+    void* p_c;
+    float epsilon;
+    ck_tile::index_t batch_size;
```

Comment:
Currently the batch size will affect the GridSize of the Gemm kernel which will determine how the grid of matrix A and B will reiterate. I think we could keep it now otherwise we also need to code every one of them to 1.

Reply:
OK, we may leave it, but let's rename it as `kbatch`. Batched Gemm may need different approach.

Reply:
Yes, I will put it as kbatch. The basic vanilla GEMM could support multi-batch but we tend to use the batched gemm to run the multi-batch scenario.